### ISSUE TITLE
Editor reformats code, and its formatting doesn't match the style IntelliJ uses

### ISSUE THREAD

**Issue by zsmb13** [JetBrains/kotlin-playground#15]
### Example, chained method calls:

Original code:

```kotlin
val allowedEntrance = people
        .filter { it.age &gt;= 21 }
        .map { it.name }
        .take(5)
```

Reformatted by the editor:

```kotlin
val allowedEntrance = people
.filter { it.age >= 21 }
.map { it.name }
.take(5)
```

### Example, lambda braces:

Original:

```kotlin
args.forEach { arg -&gt;
    println(arg)
}
```

Reformatted:

```kotlin
args.forEach { arg ->
              println(arg)
             }
```

**Comment by AlexanderPrendota** [comment]
@zsmb13 Yes, I know, but It's not main task we have. Unfortunately it's Codemirror issue in [clike mode](https://github.com/codemirror/CodeMirror/blob/master/mode/clike/clike.js). If you have any time try to help us. We will be happy.

**Comment by AlexanderPrendota** [comment]
Fixed since commit [`e338ab2`](https://github.com/codemirror/CodeMirror/commit/e338ab23b69dd8aebe0a976220387e0ac187d107)
